.github/workflows/ci.yml

```diff
@@ -0,0 +1,35 @@
+name: CI
+
+on:
+  [pull_request, push]
+
+jobs:
+  build:
+    name: Build
+    strategy:
+      matrix:
+        os: [ 'ubuntu-latest', 'macos-latest' ]
+        ocaml-version:
+          - 4.10.1
+          - 4.11.1
+    runs-on: ${{ matrix.os }}
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Setup OCaml ${{ matrix.ocaml-version }}
+        uses: avsm/setup-ocaml@v1
+        with:
+          ocaml-version: ${{ matrix.ocaml-version }}
+        
+      - name: Install SATySFi dependencies
+        run: |
+          opam repository add satysfi-external https://github.com/gfngfn/satysfi-external-repo.git
+          opam update
+          opam pin add -y --no-action satysfi .
+          opam install -y --deps-only satysfi
```

Comment:
```suggestion
          opam repository add satysfi-external https://github.com/gfngfn/satysfi-external-repo.git
          opam update
          opam pin add satysfi.dev . --no-action
          opam depext satysfi --yes --with-doc --with-test
          opam install . --deps-only --with-doc --with-test
```

Reply:
I don't know OCaml CI fashion very well 😇  
Do @na4zagin3 and @gfngfn think it's better than the current one?

Reply:
It depends what you mean by better.

Reply:
Except for the depext part, there are no other major and/or minor changes. (See here for a description of what depext is.)
https://github.com/avsm/setup-ocaml#what-is-the-difference-between-opam-dependencies-and-depext-dependencies
Note that GitHub Actions VM images are relatively unstable. With that in mind, even if depext is not currently needed, it makes sense to include it as part of our workflows.

Reply:
OK, thanks for your explanation to me 🙇 
I'll follow your suggestion and rebase these commit.

Reply:
I work with OCaml Labs and GitHub to work on Actions itself and a lot of infrastructure stuff like setup-ocaml, so feel free to ping me if you have any questions :)